src/modules/l0-server.c

```diff
@@ -591,23 +591,25 @@ static ksnLNullData* ksnLNullClientRegister(ksnLNullClass *kl, int fd, const cha
     return kld;
 }
 
-void _send_subscribe_events(ksnetEvMgrClass *ke, const char *name,
-        size_t name_length) {
+void _send_subscribe_events(ksnetEvMgrClass *ke, const char *payload,
+        size_t payload_length) {
 
     // Send Connected event to all subscribers
     teoSScrSend(ke->kc->kco->ksscr, EV_K_L0_CONNECTED,
-        (void *)name, name_length, 0);
+        (void *)payload, payload_length, 0);
 
     // Add connection to L0 statistic
     ke->kl->stat.visits++; // Increment number of visits
 
+    // This is important code for admin-panel
+    //
     // Send "new visit" event to all subscribers
-    size_t vd_length = sizeof(ksnLNullSVisitsData) + name_length;
-    ksnLNullSVisitsData *vd = malloc(vd_length);
-    vd->visits = ke->kl->stat.visits;
-    memcpy(vd->client, name, name_length);
-    teoSScrSend(ke->kc->kco->ksscr, EV_K_L0_NEW_VISIT, vd, vd_length, 0);
-    free(vd);
+    // size_t vd_length = sizeof(ksnLNullSVisitsData) + payload_length;
+    // ksnLNullSVisitsData *vd = malloc(vd_length);
+    // vd->visits = ke->kl->stat.visits;
+    // memcpy(vd->client, payload, payload_length);
+    // teoSScrSend(ke->kc->kco->ksscr, EV_K_L0_NEW_VISIT, vd, vd_length, 0);
+    // free(vd);
```

Comment:
Commented code makes dozens of noise in search results
Its very good practice to remove code instead of commenting (ifdef'ing) it

Reply:
@chernikovdmitry  I agree. I will make a separate callback for this event (EV_K_L0_NEW_VISIT)